### ISSUE TITLE
ValueError: Namespace Gtk is already loaded with version 3.0

### ISSUE THREAD

**Issue by StephenGravitt** [chr314/nautilus-copy-path#55]
Hello,

I used the net install script. Using ubuntu 22.04 and I got this error after the command: `$ nautilus -q`
```
Traceback (most recent call last):
  File "/home/stephengravitt/.local/share/nautilus-python/extensions/nautilus-copy-path.py", line 4, in <module>
    from nautilus_copy_path import NautilusCopyPath
  File "/home/stephengravitt/.local/share/nautilus-python/extensions/nautilus-copy-path/nautilus_copy_path.py", line 9, in <module>
    require_version('Gtk', '4.0')
  File "/usr/lib/python3/dist-packages/gi/__init__.py", line 117, in require_version
    raise ValueError('Namespace %s is already loaded with version %s' %
ValueError: Namespace Gtk is already loaded with version 3.0
Initializing nautilus-dropbox 2019.02.14
```

**Comment by StephenGravitt** [comment]
I came up with a temporary work-around and still hope for a better solution down the line.

I admittedly know nothing of GTK but with some half-hearted research I realized fixing the dependency is something I'm not up for trying at first. I welcome feedback if this is a better route to go.

I resolved to check previous releases as to when GTK was used and found release for [version 1.8.0](https://github.com/chr314/nautilus-copy-path/releases/tag/1.8.0). 

Here is what I did to install the previous version and resolve my issues:

_Note: that I used the "net-installer.sh" to install and at the time of writing this the default install location is `~/.local/share/nautilus-python/extensions`_

1. Deleted nautilus-copy-path.sh and nautilus-copy-path directory from install location 
2. updated line 8 of the installer (which has a curl & wget command to download) to: `wget -O nautilus-copy-path.tar.gz https://github.com/chr314/nautilus-copy-path/archive/refs/tags/1.8.0.tar.gz`
3. `bash net-installer.sh`
4. Basque in the glory that is "Copy Path"



Thank you nautilus-copy-path team!
